### ISSUE TITLE
Brainstorming around Business rules

### ISSUE THREAD

**Issue by mohierf** [shinken-monitoring/mod-webui#240]
What should the WebUI do with business rules: 
- [ ] display business rules
- [ ] help in building business rules ?

**Comment by Simage** [comment]
Display is a definite.. possibly in such a way that you can see what part of the rule is causing the failure and which hosts and/or services are referenced.  

While I think it may be nice to build and manage business rules and other config items from within the webui I wonder if that's perhaps a better task for a plug in. .. a plugin/module should be able to register it's own url using the internal appreference I believe....

**Comment by maethor** [comment]
You are absolutely right @Simage. It's not the case for know, but we could make it work to lighten the webui.

**Comment by mohierf** [comment]
+1. Thansk @Simage for your comments

**Comment by Simage** [comment]
@maethor Unless I'm missing something in my read, that's simply loading the plugins isn't it?

**Comment by maethor** [comment]
That's loading the plugins… for each module (the loop on line 314). So each module can brings it's pages.

Maybe I didn't understand your question?
